dockerBasedBuild/dockerBasedBuild.common.psm1

```diff
@@ -509,3 +509,32 @@ function script:logerror([string]$message) {
     #reset colors for older package to at return to default after error message on a compilation error
```

Comment:
`to at return to default` maybe should be changed to `to default at return`.
Just curious, why do we need to reset the color?

Reply:
This is a cut and paste from build.psm1.  I took the code because it works...

Reply:
Thinking about it.  I suspect something other than PowerShell changed the console color and didn't reset it and this code was added to "fix" this issue.  I remember making this call in a script I wrote (not build.psm1.)